### PR TITLE
Implementation of functional Search component

### PR DESCRIPTION
**Pull request by bflows**
I created a functional Search component that fetches users data from an API, filter users, and map them in the list.

### CODE REVIEW

**Review by Ephraim-Winston — COMMENTED** [review]
Impeccable work my friend. One suggestion  I would make (just for best practice) is to add catch() for error handling to your fetch requests. I assumed it wasn't needed but it was the only note I saw. Keep but the good work.

**Review by Ephraim-Winston — COMMENTED** [review]
Impeccable work my friend. One suggestion I would make (just for best practice) is to add catch() for error handling to your fetch requests in searchBar.jsx. I assumed it wasn't needed but it was the only note I saw. Keep but the good work.

**Review by Ephraim-Winston — APPROVED** [review]

File: dev-duckies-sm-proj/src/pages/search/SearchResultsList/SearchResultsList.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  box-shadow: var(--box-shadow);
+  border-radius: 10px;
+  margin-top: 1rem;
+  max-height: 300px;
+  overflow-y: auto;
+  color: var(--primary-text-color);
+}
+
+/* width */
+::-webkit-scrollbar {
```
Comment (Jose-Lema) [inline dev-duckies-sm-proj/src/pages/search/SearchResultsList/SearchResultsList.css:15]: I forget you can do this kind of stuff with CSS files. Great job!

**Review by Jose-Lema — APPROVED** [review]
I didn't leave as many comments since I did not fully understand some of the CSS stuff you were doing. Wonderful job though!

**Review by FrancescaImmediato — APPROVED** [review]

**Pull request merged**